.gitmodules

```diff
@@ -1,4 +1,3 @@
 [submodule "lib/rpc_spec"]
 	path = lib/rpc_spec
 	url = https://github.com/smartdevicelink/rpc_spec.git
-	branch = master
```

Comment:
Please keep the branch property in this file. This file should be unchanged in the end.

Reply:
Well, I have restored the .gitmodules. Thanks.